modules/shapefile/src/lib/parsers/parse-dbf.ts

```diff
@@ -290,13 +290,14 @@ function parseRow(
   fields: DBFField[],
   textDecoder: TextDecoder
 ): {[key: string]: any} {
-  const out = {};
+  const out = {} as {
+		[key: string]: string | number | boolean | null;
+	};
```

Comment:
Could use use spaces, not tabs, for indentation to be consistent with the rest of the codebase? Otherwise looks good